### PR TITLE
Changes in signup texts

### PR DESCRIPTION
**Pull request by KamilWo**
Texts like: 'Imię i nazwisko' or 'Przejdź do logowania' are overwriting original texts in english 
and will prevent translation. In the future there is a need for preparing proper translation.

### CODE REVIEW

File: addons/bestja_frontend_ucw/views/login_signup.xml [hunk 1]
```diff
@@ -48,11 +48,22 @@
                 </div>
             </div>
         </template>
+        <template id="name_placeholder" inherit_id="auth_signup.fields">
+            <div class="form-group field-name" position="replace">
+                <div class="form-group field-name">
+                    /* This change prevent original translation, should be done by translation */
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/views/login_signup.xml]: This doesn’t look like a proper XML comment…

File: addons/bestja_frontend_ucw/views/login_signup.xml [hunk 2]
```diff
@@ -66,5 +77,17 @@
                 </div>
             </div>
         </template>
+        <template id="back_to_login" inherit_id="auth_signup.signup">
+            <div class="clearfix oe_login_buttons" position="replace">
+                <div class="clearfix oe_login_buttons">
+                    <div class="pull-right back_to_login">
+                        <p>Masz już konto?</p>
+                        /* This change prevent original translation, should be done by translation */
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/views/login_signup.xml]: This doesn’t look like a proper XML comment…

File: addons/bestja_frontend_ucw/views/login_signup.xml (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/views/login_signup.xml]: Fixed, thank you.

File: addons/bestja_frontend_ucw/views/login_signup.xml (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/views/login_signup.xml]: Fixed, thank you.

**Comment by KrzysiekJ** [comment]
Merged in c8dc56c.

**Pull request closed without merging**